src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+// Actions
+const CHECK_STATUS = 'bookstore/CHECK_STATUS';
+
+// Reducer
+export default function reducer(state = [], action) {
+  switch (action.type) {
+    case 'CHECK_STATUS':
+      return 'Under construction';
```

Comment:
- [ ] Nice work setting up the reducer. The switch `case` should be the variable `CHECK_STATUS` and not the string `'CHECK_STATUS'`. Kindly implement this. 👍